html/changelog.html

```diff
@@ -55,6 +55,46 @@
 		
 		-->
 		<div class="commit sansserif">
+			<h2 class="date">10 March 2015</h2>
+			<h3 class="author">Ahammer18 updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="rscadd">Adds the Slimecake and Slimecake slice.</li>
+			</ul>
+			<h3 class="author">AnturK updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="rscadd">Non-player Alien Queens/Drones now spread weeds. Can also lay eggs when enabled.</li>
+			</ul>
+			<h3 class="author">Incoming5643 updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="rscadd">Nanotrasen scientists have recently reported mutations within the pink family line of slimes. Station scientists are encouraged to investigate.</li>
+			</ul>
+			<h3 class="author">Paprika updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="tweak">Reworked reskinning and renaming guns, and added the ability to reskin the bartender's shotgun. Click it with an empty active hand to reskin. Warning, you can only do this once! Rename by hitting it with a pen. This also works with miner shotguns now.</li>
+			</ul>
+			<h3 class="author">RemieRichards updated:</h3>
+			<ul class="changes bgimages16">
+				<li class="wip">Embedding from explosions has been tweaked, they will always throw objects fast enough to get embedded if they can.</li>
+				<li class="tweak">Damage taken when embedded objects fall out has been increased.</li>
```

Comment:
Oh well, I've already done that technicallu. that line in the entry is false, nothing changed so it really shouldn't be in the changelog.

Unless you want to mention *some* embedding weapons now have unique damage values.